pom.xml

```diff
@@ -1288,6 +1290,7 @@
     <module>plugins/maven</module>
     <module>boms</module>
     <module>boms/bom</module>
+    <module>boms/bom-certified</module>
```

Comment:
@kenfinnigan from IRC... apparently I need to adjust this some.

While we have a way to do community-only builds (`-Dswarm.product.build`) I think I need to add a profile to also only include stuff if a product build.

I think a profile activated by swarm.product.build (as opposed activated by it's lack) to include exactly bom-certified.

I think preventing bom-certified from being released during community releases would help prevent confusion in the community, since we're dealing with cross-versioning of things.